llama.h

```diff
@@ -521,7 +521,8 @@ extern "C" {
                              int   text_len,
                      llama_token * tokens,
                              int   n_max_tokens,
-                            bool   add_bos);
+                            bool   add_bos,
+                            bool   allow_special_tokens);
```

Comment:
Let's use the shorter arg name `special`: 

```suggestion
                            bool   special);
```

Add comment above to explain it's function